assets/prometheus-k8s/rules.yaml

```diff
@@ -274,6 +274,8 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m]))
         BY (pod, namespace)
       record: pod:container_cpu_usage:sum
+    - expr: sum(kube_pod_container_status_last_terminated_reason{reason="OOMKilled",job="kube-state-metrics"})
+      record: kube_pod_container_status_last_terminated_reason:oom_total:sum
```

Comment:
not sure about the metric name

Reply:
Yep. This should be something like `namespace_pod_container:kube_pod_container_status_oom_total:max`
`sum` is mostly implicit and the `by` is the aggregation that's put to the front.